templates/types/streaming/nextjs/app/components/ui/chat/chat-message/markdown.tsx

```diff
@@ -82,10 +68,8 @@ export default function Markdown({
   const { backend } = useClientConfig();
 
   return (
-    <MemoizedReactMarkdown
-      className="prose dark:prose-invert prose-p:leading-relaxed prose-pre:p-0 break-words custom-markdown"
-      remarkPlugins={[remarkGfm, remarkMath]}
-      rehypePlugins={[rehypeKatex as any]}
+    <MarkdownUI
+      content={processedContent}
       components={{
```

Comment:
move to chat-ui as is (we can later add a possibility to overwrite it) - here we just want the sandbox feature added